src/prelude.rs

```diff
@@ -6,7 +6,8 @@ pub use crate::flash::FlashExt as _stm32f3xx_hal_flash_FlashExt;
 pub use crate::gpio::GpioExt as _stm32f3xx_hal_gpio_GpioExt;
 pub use crate::hal::prelude::*;
 pub use crate::rcc::RccExt as _stm32f3xx_hal_rcc_RccExt;
-pub use crate::time::U32Ext as _stm32f3xx_hal_time_U32Ext;
+pub use embedded_time::duration::*;
+pub use embedded_time::rate::*;
```

Comment:
If I'm still up to date, this way of writing it is preferred:
`pub use embedded_time::{duration::*, rate::*};`

Reply:
I believe I removed these in the latest commit.